src/api/TasksApiV1.ts

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
I really agonised over whether to ask this, but I care a lot about consistency, so I'm curious about the choice of lower-case `api/` in the folder names in `src/` and `tests/`.

This isn't a request to change it... just wondering what prompted the decision.

(I note we have `lib` and `ui` already.. and `config` was at some point renamed to `Config`.)

Reply:
Renamed directories to `Api/`.